### PR TITLE
[dev2] [Part 3] Configuration files: settings.yml

### PR DESCRIPTION
**Pull request by franramirez688**
Examples about  `settings_user.yml`: TBD

MERGE FIRST -> https://github.com/conan-io/docs/pull/2959

### CODE REVIEW

File: examples/config_files/settings/customizing_settings.rst [hunk 1]
```diff
@@ -0,0 +1,146 @@
+.. _examples_config_files_customizing_settings:
+
+Customizing settings
```
Comment (memsharded) [inline examples/config_files/settings/customizing_settings.rst:3]: I am missing the ``settings_user.yml`` new 2.0 feature, at least a place holder to fill it later.

**Review by memsharded — APPROVED** [review]

**Comment by franramirez688** [comment]
@czoido @memsharded I think that the new feature `settings_user.yml` is worth of having its own example.

**Review by memsharded — APPROVED** [review]

File: reference/commands/config.rst [hunk 2]
```diff
@@ -20,14 +20,17 @@ conan config home
...
 The ``conan config home`` command returns the path of the Conan home folder.
-See the config files documentation for more information.
-
 
 .. code-block:: bash
 
     $ conan config home
 
 
+.. seealso::
```
[the remark this answers is not in the document]
Comment (franramirez688) [reply reference/commands/config.rst:29]: Actually, I did not know where to put it... It's related to all the commands (and sub-commands) so I thought that it's better to put it in first place but yes, it's related to all the sub-commands.

File: reference/config_files/settings.rst [hunk 3]
```diff
@@ -0,0 +1,481 @@
...
+.. seealso::
+
+    - :ref:`creating_packages_configure_options_settings`
+
+
+Operating systems
+-----------------
+
+``baremetal`` operating system (introduced in Conan 1.43) is a convention meaning that the binaries run directly
+on the hardware, without a operating system or equivalent layer. This is to differentiate to the ``None`` value,
```
Comment (czoido) [inline reference/config_files/settings.rst:149]:
```suggestion
``baremetal`` operating system is a convention meaning that the binaries run directly
on the hardware, without a operating system or equivalent layer. This is to differentiate to the ``None`` value,
```

File: reference/config_files/settings.rst [hunk 4]
```diff
@@ -0,0 +1,481 @@
...
+which is associated to the "this value is not defined" semantics. The ``baremetal`` is a common name convention for
+embedded microprocessors and microcontrollers code. It is expected that users might customize the space inside the
+``baremetal`` setting with further subsettings to specify their specific hardware platforms, boards, families, etc.
+At the moment (Conan 1.43) the ``os=baremetal`` value is still not used by Conan builtin toolchains and helpers,
```
Comment (czoido) [inline reference/config_files/settings.rst:153]:
```suggestion
At the moment the ``os=baremetal`` value is still not used by Conan builtin toolchains and helpers,
```

**Review by czoido — APPROVED** [review]

**Review by RubenRBS — APPROVED** [review]
👍

**Pull request merged**